### PR TITLE
Rename Message.from_ to Message.sender

### PR DESCRIPTION
**Pull request by thesharp**

### CODE REVIEW

File: botogram/objects/mixins.py [hunk 1]
```diff
@@ -140,6 +140,12 @@ def send_sticker(self, sticker, reply_to=None, extra=None, notify=True):
 class MessageMixin:
     """Add some methods for messages"""
 
+    @property
+    @utils.deprecated("Message.from_", "1.0",
+                      "Rename property to Message.sender", back=-3)
+    def from_(self):
+        return self.sender
+
```
Comment (pietroalbini) [inline botogram/objects/mixins.py]: Can this be on the main Message object? It's an attribute of it after all.

File: docs/api/telegram.rst [hunk 2]
```diff
@@ -508,6 +508,17 @@ about its business.
...
+   .. py:attribute:: sender
+
+      The sending :py:class:`~botogram.User` of the message. Note the trailing
+      underscore, needed due to 'from' being a python keyword.
+
+      *This attribute can be None if it's not provided by Telegram.*
+
+   .. versionchanged:: 0.2
+
+      Before it was called ``from_``.
```
Comment (pietroalbini) [inline docs/api/telegram.rst]: Wrong indent :)

File: docs/api/telegram.rst [hunk 3]
```diff
@@ -515,6 +526,8 @@ about its business.
 
       *This attribute can be None if it's not provided by Telegram.*
 
+   .. deprecated:: 0.2 It will be removed in botogram 1.0
+
```
Comment (pietroalbini) [inline docs/api/telegram.rst]: Wrong indent :)

Comment (thesharp) [inline docs/api/telegram.rst]: Are you sure? It has three spaces.

Comment (pietroalbini) [inline docs/api/telegram.rst]: Yes, but it belongs to the attribute, which is double indented.

```
class
   attribute
      deprecated
```

File: docs/api/telegram.rst [hunk 4]
```diff
@@ -508,6 +508,17 @@ about its business.
...
+   .. py:attribute:: sender
+
+   The sending :py:class:`~botogram.User` of the message. Note the trailing
+   underscore, needed due to 'from' being a python keyword.
+
+   *This attribute can be None if it's not provided by Telegram.*
+
+   .. versionchanged:: 0.2
+
+   Before it was called ``from_``.
```
Comment (pietroalbini) [inline docs/api/telegram.rst]: Wrong indentation... Again! (one more level)

**Comment by pietroalbini** [comment]
Thank you!

**Pull request merged**